grobid-core/src/main/java/org/grobid/core/main/LibraryLoader.java

```diff
@@ -126,7 +129,7 @@ public boolean accept(File dir, String name) {
                     try {
                         System.load(wapitiLibFiles[0].getAbsolutePath());
                     } finally {
-                        if (GrobidProperties.getGrobidCRFEngine() == GrobidCRFEngine.DELFT) {
+                        if (CollectionUtils.containsAny(distinctModels, Arrays.asList(GrobidCRFEngine.DELFT))) {
```

Comment:
I guess always using `singletonList` was too boring :D

Reply:
singletonList is for single is for single element's lists ;-)

Reply:
Isn't the right hand side containing just one element like above, i.e. `GrobidCRFEngine.DELFT`? I hadn't had my coffee yet though :D
(I only noticed that there is a difference, it doesn't really matter)